test/dynamo/test_repros.py

```diff
@@ -2625,6 +2626,20 @@ def fn(x):
         res = opt_fn(x)
         self.assertTrue(same(ref, res))
 
+    def test_call_dict_kwargs(self):
+        class Foo(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.a = torch.randn(4)
+
+            def forward(self, x):
+                outputs = torch.sin(x) + torch.cos(self.a)
+                return dict(x=x, outputs=outputs)
+
+        with self.assertNoLogs(logger="torch._dynamo", level=logging.WARNING):
+            foo = torch.compile(Foo())
+            foo(torch.randn(4))
```

Comment:
Testing if output specific log is not a good idea, let's use if graph break to test this. You can add a test follow & after this:
https://github.com/pytorch/pytorch/blob/b04f86363fce9828b066ee516e78912691129b3f/test/dynamo/test_functions.py#L611-L617

All test in ```dynamo/test_functions.py``` is no graph break.

Reply:
Added `test_call_dict6`, you're right that it's much simpler